src/weblib.ts

```diff
@@ -89,3 +89,10 @@ export async function generate_transfer() {
     displayMessage('tx', `Payload: <br/> <br/> ${encoded}`, 'Unsigned Transaction')
 }
 
+export async function broadcastTransaction() {
+    const tx = getInputElement('broadcast-input');
+    const deserializeTx = StxTx.deserializeTransaction(tx)
```

Comment:
Not sure this is correct... data should be input in base64 (because that's what the other steps use) and I don't think that's what `deserializeTransaction()` expects. I think it should be:

```suggestion
    const deserializeTx = base64Deserialize(tx) as StxTx.StacksTransaction;
```

Reply:
`broadcastTransaction` expects a StacksTransaction instance so it won't work with a deserialized base64.